test/halmos/HalmosTest.sol

```diff
@@ -47,6 +51,23 @@ contract HalmosTest is SymTest, Test {
         morpho.enableLltv(lltv);
         morpho.createMarket(marketParams);
         vm.stopPrank();
+
+        // for flashLoan
+        otherToken = new ERC20Mock();
+        flashBorrower = new FlashBorrowerMock(morpho);
```

Comment:
a contract that implements `onMorphoFlashLoan()` is needed to execute flashLoan(). other mock contracts can also be used.

Reply:
What happens if we put those definitions in the constructor of the current contract ? Would it still be considered as valid targets for symbolic addresses ?

Reply:
do you mean adding onMorphoFlashLoan() in the test contract, right? yes, it will work. any addresses including the test contract will be considered as a target for symbolic addresses.

Reply:
> do you mean adding onMorphoFlashLoan() in the test contract?

I was more thinking about having those 2 lines inside `constructor() { ... }` instead of inside `function setUp() { ... }` where it is currently. I'm asking this because the line `ERC20Mock internal otherToken;` could be changed to `ERC20Mock internal otherToken = new ERC20Mock();` instead

Reply:
I see. Yes, you could do that for otherToken, but not for flashBorrower as it needs `morpho` as input.